src/main/java/org/generationcp/middleware/pojos/ims/Transaction.java

```diff
@@ -100,6 +100,10 @@
 	@Column(name = "bulk_compl")
 	private String bulkCompl;
 
+	@Basic(optional = false)
+	@Column(name = "trntype")
+	private Integer type;
```

Comment:
Instead of using Integer, you could try to declare it as a TransactionType and then use a custom converter to save the id instead of ordinal or string. See https://thoughts-on-java.org/hibernate-enum-mappings/  Create a custom mapping section. 
Please verify that this work for null values 
Same can be done with the status

Reply:
Couldn't be implemented